Bottom: 7d1529033c1ec49df02e1e5aa4d0b0a536dcdf50
Top:    87120d91e94d1b3ab49021d18c989e66bcc5876d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:27:20 +0000

Refresh of fix-board-outline-generation

---

diff --git a/src/polygon.c b/src/polygon.c
index f6ae8ab..da734f8 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2459,17 +2459,11 @@ POLYAREA *board_outline_poly (bool include_holes)
   return clipped;
 #endif
 
-  if (clipped == NULL)
-    {
-      fprintf (stderr, "clipped == NULL in board_outline_poly\n");
-      return clipped;
-    }
-
   /* Now we just need to work out which pieces of polygon are inside
      and outside the board! */
 
-  /* If there is only one piece, return that */
-  if (clipped->f == clipped)
+  /* If there is no result, or only one piece, return that */
+  if (clipepd == NULL || clipped->f == clipped)
     return clipped;
 
   /* WARNING: This next check is O(n^2), where n is the number of clipped
